### PR TITLE
Bug/prefs fix

### PR DESCRIPTION
**Pull request by abrenoch**
Not much here now, @ninovanhooff 's recent PR took care of most of the key name references (which this mostly was previously).

Just a few errant items!

### CODE REVIEW

File: mobile/src/main/java/com/abrenoch/hyperiongrabber/mobile/SettingsActivity.java [hunk 1]
```diff
@@ -51,12 +49,12 @@ private static void bindPreferenceSummaryToValue(Preference preference) {
...
 
+        Preferences prefs = new Preferences(preference.getContext());
+
         // Trigger the listener immediately with the preference's
         // current value.
         sBindPreferenceSummaryToValueListener.onPreferenceChange(preference,
-                PreferenceManager
-                        .getDefaultSharedPreferences(preference.getContext())
-                        .getString(preference.getKey(), ""));
+                prefs.getString(preference.getKey(), ""));
```
Comment (ninovanhooff) [inline mobile/src/main/java/com/abrenoch/hyperiongrabber/mobile/SettingsActivity.java]: I would slightly prefer this change to be reversed and the corresponding method to be removed from `Preferences.kt`. The reason being 
- The net result will be exactly the same
- It introduces a way to work with strings as keys in Preferences.kt, which counters it's purpose
- It introduces asymmetry: Preferences.kt is used for reading values, but SharedPreferences is used for writing values

File: common/src/main/java/com/abrenoch/hyperiongrabber/common/util/Preferences.kt [hunk 2]
```diff
@@ -16,6 +16,10 @@ class Preferences(context: Context) {
         return preferences.getString(key(keyResourceId), default)
     }
 
+    fun getString(key: String, default: String? = null): String? {
```
Comment (ninovanhooff) [inline common/src/main/java/com/abrenoch/hyperiongrabber/common/util/Preferences.kt]: I would suggest reverting this change. See other comment

File: mobile/src/main/java/com/abrenoch/hyperiongrabber/mobile/SettingsActivity.java (hunk 1 above)
Comment (abrenoch) [reply mobile/src/main/java/com/abrenoch/hyperiongrabber/mobile/SettingsActivity.java]: Sounds alright to me - agree with the reasoning too!

**Comment by abrenoch** [comment]
Got those items taken care of!

**Pull request merged**